### DISCUSSION TITLE
tinydb in production

### DISCUSSION THREAD

**Discussion by jgarte** [msiemens/tinydb#433] [Q&A]
Does anyone happen to know any apps that use tinydb in production?

**Comment by msiemens** [comment]
Depends on what you mean by production 🙂 If you mean some sort of web service that uses TinyDB as its primary storage backend, I don't know of any. But there are a few CLI tools and other projects that use TinyDB, some of which are listed here: https://github.com/msiemens/tinydb/wiki/Used-By
